### ISSUE TITLE
Charging Text Always shows "Charging Slowly"

### ISSUE THREAD

**Issue by Michael-Bell** [CypherOS/device_lge_g4-common#1]
Using any charger (AC, QC 2.0, USB) always shows "Charging Slowly" on the lockscreen. Charges at the same rates as stock on all chargers, despite text.

**Comment by Sirhc95** [comment]
Doesn't do that for me unless i use a generic charger not meant for my device. Have you tried the factory charger

**Comment by Michael-Bell** [comment]
Yes. I've been switching between stock rom and cypher. Cypher will display "Charging Slowly" under the same conditions that stock displays "Charging Rapidly"

Running CypherOS 3.1.4, Release ID CNRC1013.

**Comment by Sirhc95** [comment]
Try updating to the latest version ?

**Comment by Michael-Bell** [comment]
On CypherOS  3.6 Release ID CR.NR22.36.R1-0223 it now shows 
```
Charging slowly (40 mins until full)

1mA/h / 5V
```

When the battery is 15% full. This shows me it is charging at rapid speeds, where stock rom would display `Charging Rapidly`.

**Comment by Sirhc95** [comment]
This is either a failure at detecting your ma/h from your charger or simply a protection from overheating your phone. I'm not the maintainer of the g4 so i can't give a full solution

**Comment by astraujums** [comment]
The same bug happens for LineageOS 14.1-20171107-NIGHTLY-h815, so it seems to be "inherited".
